setup/upgrade_major.rst

```diff
@@ -108,11 +108,15 @@ done!
             <!-- ... -->
 
             <php>
-                <env name="SYMFONY_DEPRECATIONS_HELPER" value="weak"/>
+                <env name="SYMFONY_DEPRECATIONS_HELPER" value="max[total]=999999"/>
             </php>
         </phpunit>
 
-    (you can also execute the command like ``SYMFONY_DEPRECATIONS_HELPER=weak phpunit``).
+    You can also execute the command like:
+
+    .. code-block:: terminal
+
+        $ SYMFONY_DEPRECATIONS_HELPER=max[total]=999999 phpunit
```

Comment:
@javiereguiluz do we need to add „php“ in front of „phpunit“?

Reply:
In the docs, we don't use "pure PhpUnit" but PhpUnitBridge ... so we don't have examples of executing the regular PhpUnit script. However, thanks to the latest Symfony recipes, you don't have to do `./vendor/bin/simple-phpunit` because `./bin/phpunit` is equivalent (that's what we show in https://symfony.com/doc/current/testing.html for example).

Maybe we can use `./bin/phpunit` too here?

Reply:
Then, we should use `php ./bin/phpunit` to have Windows compatibility

Reply:
Yes please

Reply:
@OskarStark @wouterj done :)